infrastructure/demo-release.bat

```diff
@@ -1,5 +0,0 @@
-CALL cd ..
```

Comment:
I actually meant nor toremove this file,
But make it 
```release.bat demo-covid19livebot```
and create a seperate one for prod
same way to be prod.release.bat

Reply:
done